docs/sources/enterprise/http_api.md

```diff
@@ -0,0 +1,72 @@
++++
+title = "Enterprise HTTP APIs"
+description = "Grafana Enterprise APIs"
+keywords = ["grafana", "enterprise", "api"]
+aliases = ["/docs/grafana/latest/enterprise/http_api/"]
+type = "docs"
+[menu.docs]
+name = "HTTP APIs"
+parent = "enterprise"
+weight = 1000
++++
+
+# Enterprise HTTP APIs
```

Comment:
Add an intro here and to each h2. It can be short, but it should set the context so the reader knows what to expect in the document or section.